networking/p2p/src/main/java/tech/pegasys/teku/networking/p2p/libp2p/gossip/GossipHandler.java

```diff
@@ -102,8 +121,8 @@ public void gossip(Bytes bytes) {
       // We've already gossiped this data
       return;
     }
-
     LOG.trace("Gossiping {}: {} bytes", topic, bytes.size());
+    bytesSentCounter.inc(bytes.size());
```

Comment:
This will likely count things that the libp2p level actually sees as a duplicate and so doesn't send.  A lot of that will be caught by the `processedMessages` at this level but not all.

We also won't be counting any messages that we sent as part of rebroadcasting gossip we immediately found to be valid.

Reply:
yep mine sits on zero as a mainnet node with no validators... is it an adequate starting point, or is it not worth having? I can just leave it at 0 if we're not happy with missing all of that?

Reply:
I think given how misleading this is we wouldn't want to make it a metric at all.